smu/dataset_pb2.pyi

```diff
@@ -0,0 +1,1373 @@
+"""
```

Comment:
Did you intentionally include this generated file?

Reply:
No, thanks for the catch!